lib/getWhitelist.js

```diff
@@ -0,0 +1,26 @@
+function getWhitelist() {
+  let whitelist = [];
+  try {
+    whitelist = require("./../whitelist.json");
```

Comment:
I might add an assertion to make sure it's an array here, rather than having to do it on line 18 — that could simplify things a bit.